### PR TITLE
feat(kubernetes): allow a copy from running infrastructure into deplo…

### PR DESCRIPTION
**Pull request by danielpeach**
…y manifest stage

Link to open modal:
copy_button

Modal:
copy_options

### CODE REVIEW

**Comment by danielpeach** [comment]
- Mostly captures the functionality of the `DeployInitializer`, but with fewer opinions (just copies a running manifest). 
- Like the `DeployInitializer`, it only offers the most recent server group in a cluster
- Offers other Kubernetes kinds (deployments, services, etc).
- Doesn't offer a replicaSet if it's managed by a deployment

File: app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+export interface IManifestOption {
+  account: string;
+  location: string;
+  name: string;
+  kind: string;
+  data: any;
+  key: string;
+}
+
+const LAST_APPLIED_CONFIGURATION = 'kubectl.kubernetes.io/last-applied-configuration';
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx]: :+1:

File: app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+/*
+ * A modal that allows a user to copy a running Kubernetes resource.
+ **/
+export class ManifestCopier extends React.Component<IManifestCopierProps, IManifestCopierState> {
+  public static getDerivedStateFromProps = (props: IManifestCopierProps): IManifestCopierState => {
+    let manifests: IManifestOption[] = [];
+
+    const serverGroups: IServerGroup[] = props.application.getDataSource('serverGroups').data.filter(
```
Comment (lwander) [inline app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx]: Why only server groups?

Comment (danielpeach) [reply app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx]: What do you mean? They're just handled a little differently, but the other kinds are included...

Comment (lwander) [reply app/scripts/modules/kubernetes/src/v2/pipelines/stages/ManifestCopier.tsx]: Sorry got sidetracked halfway through. I second (from your todo) using the account map to derive kinds to load

**Pull request merged**